src/runtime_src/xrt/device/hal2.h

```diff
@@ -76,7 +76,7 @@ class device : public xrt::hal::device
     void *hostAddr = nullptr;
     size_t size = 0;
     size_t offset = 0;
-    xclBOKind kind;
+    int kind;
```

Comment:
Can we remove kind from this internal struct?  It doesn't look like kind is used any where.